components/brave_referrals/browser/brave_referrals_service.h

```diff
@@ -49,8 +51,6 @@ class BraveReferralsService {
   std::string BuildReferralFinalizationCheckPayload() const;
   void FetchReferralHeaders();
```

Comment:
We shouldn't need `FetchReferralHeaders` now, right?

Reply:
Removed.